scripts/prepare.js

```diff
@@ -0,0 +1,42 @@
+'use strict';
+const fs = require('fs');
+const crypto = require('crypto');
+const request = require('request');
+
+if (!fs.existsSync('server')){
+    fs.mkdirSync('server');
+}
+
+if (!fs.existsSync('analyzers')){
+    fs.mkdirSync('analyzers');
+}
+
+downloadIfNeeded('https://repox.sonarsource.com/sonarsource-public-builds/org/sonarsource/sonarlint/core/sonarlint-language-server/2.15.0.780/sonarlint-language-server-2.15.0.780.jar', 'server/sonarlint-ls.jar');
+downloadIfNeeded('http://repo1.maven.org/maven2/org/sonarsource/javascript/sonar-javascript-plugin/3.1.0.5111/sonar-javascript-plugin-3.1.0.5111.jar', 'analyzers/sonarjs.jar');
+downloadIfNeeded('http://repo1.maven.org/maven2/org/sonarsource/php/sonar-php-plugin/2.10.0.2087/sonar-php-plugin-2.10.0.2087.jar', 'analyzers/sonarphp.jar');
+downloadIfNeeded('http://repo1.maven.org/maven2/org/sonarsource/python/sonar-python-plugin/1.8.0.1496/sonar-python-plugin-1.8.0.1496.jar', 'analyzers/sonarpython.jar');
+
+function downloadIfNeeded(url, dest) {
```

Comment:
This will download the sha1 every time. But the sha1 is not expected to change, normally. So I think that's a waste. 

I think it would be better to simply check if the file exists. Download if it doesn't. And then of course, when downloading a jar, it's good to download the sha1 as well and verify the checksum.